### PR TITLE
💱 Add Johnletey's currencies and banknote icons

### PR DESCRIPTION
**Pull request by delnyn**
Create banknote.svg, bitcoin.svg, euro.svg, pound-sterling.svg, indian-rupee.svg, jersey-pound.svg, russian-ruble.svg, swiss-franc.svg, and update tags.json

### CODE REVIEW

**Comment by delnyn** [comment]
would help for #93

**Comment by locness3** [comment]
Use `Co-Authored-By` when merging ;)

**Comment by delnyn** [comment]
where?

File: icons/euro.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  xmlns="http://www.w3.org/2000/svg"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M16 10.5H6" />
+  <path d="M15 13.5H6" />
+  <path d="M18 17.545c-.898.675-1.993.955-3.118.955-3.235 0-6.382-3-6.382-6.5s3.147-6.5 6.382-6.5c1.125 0 2.22.25 3.118.93" />
```
Comment (ericfennis) [inline icons/euro.svg:14]:
```suggestion
  <path d="M4 10h12" />
  <path d="M4 14h9" />
  <path d="M19 6a7.7 7.7 0 00-5.2-2A7.9 7.9 0 006 12c0 4.4 3.5 8 7.8 8 2 0 3.8-.8 5.2-2" />
```

I refactored the euro sign a bit. So it matches more with the rest. (top left is the original)
image

File: icons/banknote.svg [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<svg
```
Comment (ericfennis) [inline icons/banknote.svg:1]: We need also refactor this icon, it's violating the 2px spacing.

**Comment by delnyn** [comment]
refactored it! 
image
100% 
banknote

**Comment by ericfennis** [comment]
@delnyn Awesome!

**Comment by ericfennis** [comment]
@delnyn What do you think of my suggestion?

**Comment by delnyn** [comment]
oh yeah @ericfennis I forgot to commit it 😁

**Comment by locness3** [comment]
> co-authored-by me or john?
> 

Depends on the circumstance

**Comment by ericfennis** [comment]
Oh shit I forgot to add john 🤦

**Pull request merged**